### PR TITLE
Managing images: reference and insecure import policies

### PR DESCRIPTION
**Pull request by miminar**
Documenting the latest changes to handling image stream tags annotated
with above policies.

### CODE REVIEW

**Comment by miminar** [comment]
@legionus for technical review.
@mfojtik for the referencePolicy part.

**Review by legionus — APPROVED** [review]
LGTM

**Comment by miminar** [comment]
@ahardin-rh could you please take a look?

File: dev_guide/managing_images.adoc [hunk 1]
```diff
@@ -103,6 +103,17 @@ refreshed (i.e., re-imported) periodically. The period is configured globally at
...
 Image Metadata] for more details.
 
+If you want to instruct docker to always fetch the tagged image from the
+integrated registry, use `--reference-policy=local`. The registry will use
+xref:../install_config/registry/extended_registry_configuration.adoc#middleware-repository-pullthrough[pull-through
+feature] to serve the image to the client and by default, the image blobs will
+be mirrored locally by the registry. As a result, they will be available in
+shorter time next time they are needed. The flag also allows for pulling from
+insecure registries without a need to supply `--insecure-registry` to the
+Docker daemon if the image stream has an xref:#insecure-registries[insecure
```
Comment (ahardin-rh) [inline dev_guide/managing_images.adoc]: You can drop the # within an xref if pointing to a section on the same page

File: dev_guide/managing_images.adoc [hunk 2]
```diff
@@ -103,6 +103,17 @@ refreshed (i.e., re-imported) periodically. The period is configured globally at
...
+annotation] or the tag has an xref:#insecure-tag-importpolicy[insecure import
```
Comment (ahardin-rh) [inline dev_guide/managing_images.adoc]: Same here. No # needed.

**Comment by ahardin-rh** [comment]
@miminar  Is this targeted for 3.5? 

Overall, this looks good. I am going make some general style edits in a follow-up PR. Thanks!

**Comment by miminar** [comment]
Thanks! Comments addressed.

@ahardin-rh yes, 3.5.

**Comment by ahardin-rh** [comment]
Thank you!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../dev_guide/managing_images.adoc#dev-guide-managing-images[Managing Images]
|Added new sections about the xref:../dev_guide/managing_images.adoc#insecure-tag-import-policy[Insecure Tag Import Policy] and xref:../dev_guide/managing_images.adoc#reference-policy[Reference Policy].
%

**Pull request merged**